frontend/lib/fhirUtils.ts

```diff
@@ -0,0 +1,124 @@
+import Client from 'fhir-kit-client';
+import { Bundle, CarePlan, CarePlanActivity, Condition, Patient, Resource, ServiceRequest, Task } from 'fhir/r4';
+
+type FhirClient = Client;
+type FhirBundle<T extends Resource> = Bundle<T>;
+
+export const BSN_SYSTEM = "http://fhir.nl/fhir/NamingSystem/bsn"
+
+export const createEhrClient = () => {
+    const baseUrl = process.env.NODE_ENV === "production"
+        ? `${typeof window !== 'undefined' ? window.location.origin : ''}/orca/contrib/ehr/fhir`
```

Comment:
the base URI `/orca/contrib/ehr/fhir` is just how we configured it, shouldn't this be an app launch parameter instead? Someone else might not have it running on `/orca`.

Or from the call to  `context`?

Reply:
We launch to orca, which redirects you to the frontend. They need to be on the same domain to share the cookie. We can only request the `context` if we know the path that orca runs on. So I suppose we either make the orca path configurable or make `/orca` path a requirement?